src/fix_set.cpp

```diff
@@ -0,0 +1,86 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#include "fix_set.h"
+
+#include "atom.h"
+#include "error.h"
+#include "set.h"
+#include "update.h"
+
+using namespace LAMMPS_NS;
+using namespace FixConst;
+
+enum{SETCOMMAND,FIXSET};     // also used in Set class
+
+/* ---------------------------------------------------------------------- */
+
+FixSet::FixSet(LAMMPS *lmp, int narg, char **arg) : Fix(lmp, narg, arg)
+{
+  if (narg < 8) error->all(FLERR, 1, "Illegal fix set command: need at least eight arguments");
+
+  nevery = utils::inumeric(FLERR, arg[3], false, lmp);
+  if (nevery <= 0) error->all(FLERR, "Fix {} Nevery must be > 0", style);
+
+  reneighbor = utils::inumeric(FLERR, arg[4], false, lmp);
+  if (reneighbor = 0 || reneighbor > 1)
```

Comment:
Accidental assignment to zero.

Should it also be `reneighbor != 1`? Not sure why it's `>1`.

Reply:
good catch - I meant it to be error if < 0 or > 1